### PR TITLE
Added Java 8 Alpine packaging

### PR DESCRIPTION
**Pull request by krufab**
Added alpine flavour, updated the Makefile accordingly and improved code of tests

### CODE REVIEW

File: Dockerfile-alpine [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# The MIT License
+#
+#  Copyright (c) 2015, CloudBees, Inc.
```
Comment (oleg-nenashev) [inline Dockerfile-alpine]: Needs to be changed unless it is a full copy-paste

File: tests/test_helpers.bash [hunk 2]
```diff
@@ -29,17 +30,52 @@ function retry {
...
+
+function clean_test_container {
+	docker kill "${SUT_CONTAINER}" &>/dev/null ||:
+	docker rm -fv "${SUT_CONTAINER}" &>/dev/null ||:
+}
+
+function is_slave_container_running {
+	sleep 1  # give time to sshd to eventually fail to initialize
+	retry 3 1 assert "true" docker inspect -f '{{.State.Running}}' "${SUT_CONTAINER}"
+}
```
Comment (oleg-nenashev) [inline tests/test_helpers.bash]:
```suggestion
}

```

**Comment by krufab** [comment]
Thanks.
It seems to me the docker-slave:alpine adds bash https://github.com/jenkinsci/docker-slave/blob/9e242b5d8efed753e1e6477d59b86eb194c267f3/Dockerfile-alpine#L38

Nevertheless, bash is required to run https://github.com/jenkinsci/docker-ssh-slave/blob/master/setup-sshd, so I think it should stay.

I'll edit the copyright and I'll come back to you

File: Dockerfile-alpine (hunk 1 above)
Comment (krufab) [reply Dockerfile-alpine]: Licence updated

**Comment by krufab** [comment]
@oleg-nenashev

**Comment by krufab** [comment]
Hi @oleg-nenashev, could you kindly have a look at this pull request? The copyright notice has been updated.

**Pull request merged**